# sys
uint8 Sys_stADMd_mp
uint8 ACCS_vSetFrmSys_mp

# chassis

chassis_input chassis_input

# planning

planning_input planning_input

# control : lateral

control_lat_debug lat_debug

# control : longitudinal

control_lon_debug lon_debug

adecu_can_debug adecu_can_debug

